lib/linguist/repository.rb

```diff
@@ -129,10 +131,9 @@ def current_tree
     end
 
     protected
-    MAX_TREE_SIZE = 100_000
```

Comment:
Wouldn't it be wiser to keep `MAX_TREE_SIZE` as a class constant so we can DRY up the class's `initialize` and `self.incremental` methods? Both methods hardcode `100_000` as the default value for their `max_tree_size` parameters…

Reply:
Thanks for the review. This makes a lot of sense.